examples/cluster/plot_cluster_iris.py

```diff
@@ -43,50 +44,51 @@
               'k_means_iris_bad_init': KMeans(n_clusters=3, n_init=1,
                                               init='random')}
 
-
+fig = plt.figure(figsize=(8, 6))
 fignum = 1
+titles = ['3 clusters', '8 clusters', '3 clusters, bad initialization']
 for name, est in estimators.items():
-    fig = plt.figure(fignum, figsize=(4, 3))
-    plt.clf()
-    ax = Axes3D(fig, rect=[0, 0, .95, 1], elev=48, azim=134)
-
-    plt.cla()
+    ax = plt.subplot(2, 2, fignum, projection='3d',
```

Comment:
actually 1.5 requires this. Issue was that I wasn't importing Axes3D because it wasn't being used directly. But I found later that this projection=3d requires Axes3D imported. So it worked after adding that import. All good now.